services/IoT/applications/libraries/python.md

```diff
@@ -266,7 +262,7 @@ By default, when you subscribe to device status, status updates are received for
 ## Handling status updates from devices
 {: #handling_device_updates}
 
-To process the status updates received by your subscriptions you need to register an event callback method. The messages are returned as an instance of the Status class.
+To process the status updates that are received by your subscriptions, you need to register an event callback method. The messages are returned as an instance of the Status class.
 
 The following properties are set for both "Connect" and "Disconnect" status events:
```

Comment:
Fixed.

Dev confirmed that the 'status.action' property determines whether a status event is of type 'Connect' or 'Disconnect'. The first list in the topic applies to all events regardless of status. The second list applies to status events with a status.action of 'Disconnect'.

Revised content as per technical clarification.